tefla/core/base.py

```diff
@@ -46,7 +46,8 @@ def __init__(self,
                loss_type='sparse_xentropy',
                weighted=False,
                label_smoothing=0.009,
-               model_name='graph.pbtxt'):
+               model_name='graph.pbtxt',
+               patience=99):
```

Comment:
@pikkaay where are you using the `patience` term?

Reply:
hard coded as 5 by mistake in early stopping fn def
 'if self.stopping_step >= 5: 
      self.early_stop = True'